docs/markdown/06-iterator_generator/05-AsyncIterator.md

```diff
@@ -0,0 +1,50 @@
+<!-- .slide:-->
+
+# Itérables Asynchrones
+
+Un itérable asynchrone est un iterable avec un itérateur dont la méthode next retourne une Promesse de {done, value} au lieu d'un objet {done, value}.
+<!-- .element: class="fragment" -->
+Un objet est considéré comme **itérable asynchrone** s'il définit le comportement qu'il aura lors de l'itération asynchrone
```

Comment:
je suis désolé, je ne suis pas sur de comprendre la phrase. Peut devrais-tu préciser cette phrase dans des notes

Reply:
j'ai voulu reproduire le format des phrases utilisées pour les itérateurs, en ajoutant asynchrone après iterable et iteration. C'était plus clair avec les itérateurs classiques  ? On peut sûrement reformuler la phrase.